starter-code/index.html

```diff
@@ -1,31 +1,57 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <meta charset="UTF-8">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="./css/main.css">
   <title>Instagram</title>
 </head>
+
 <body>
-  Instagram
+  <figure>
+    <img id="phone" src="./images/phones.png" height="450" width="300"/>
+  </figure>
+  <img id="logo" src="./images/brand.png" height="70" width="260"/>
+  </br></br>
+  <img id="frame" src="./images/frame.png" height="400" width="500"/>
+
+<div id="login">
+
+  <img id="frame" src="./images/home.png" height="20" width="20" <h4> Log In </h4>
+
+</div>
+
+  <div class="mainContainer">
 
-  Capture and Share the World's Moments
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+  <h3>Capture and Share </br>the World's Moments</h3>
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+  <p>Instagram is a <strong>fast</strong>, <strong>beautiful</strong> and <strong>fun</strong> way to share your life with friends and family.
+</br>
 
-  Oh yeah, did we mention it's free?
+Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+</br>
+  </br>Oh yeah, did we mention it's free?</p>
 
-  Download on the App Store
-  Get it on Google Play
+<img src="./images/badge-android.png" height="40" width="120"/>
+<img src="./images/badge-iphone.png" height="40" width="120"/>
+</div>
+</br>
+</br>
+    <div class="footContainer">
```

Comment:
Try nesting these links in <li> elements and turn the nav into an unordered list to provide more spacing in between each link.